drivers/rtc/Kconfig.stm32_rtc

```diff
@@ -0,0 +1,16 @@
+# Kconfig - STM32 RTC configuration options
+#
+# Copyright (c) 2018 Workaround GmbH
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config RTC_STM32
+	bool "STM32 RTC Driver"
+	depends on SOC_FAMILY_STM32
+	default n
+	select USE_STM32_HAL_RTC
+	select USE_STM32_HAL_RTC_EX
+	select USE_STM32_HAL_RCC_EX
```

Comment:
I had to add
```
	select USE_STM32_HAL_CORTEX
	select USE_STM32_HAL_PWR
	select USE_STM32_HAL_PWR_EX
```
for my build to work. Linker was missing `HAL_SYSTICK_Config()`, `HAL_NVIC_SetPriority()`, and `HAL_PWREx_GetVoltageRange()`.

Reply:
@kzyapkov : On which STM32 series did you try to build?

Reply:
Nucleo L432-KC. I could not get the alarm to fire under any conditions and I'm thinking of reworking this with LL.